charts/fees-register-api/values.yaml

```diff
@@ -17,6 +17,7 @@ java:
     IDAM_API_URL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
     ISSUER_URI: https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net/o
     OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-{{ .Values.global.environment }}.internal:8443/openam/oauth2/hmcts
+    OIDC_AUDIENCE_LIST: fees_admin_frontend,paybubble
```

Comment:
Do we need to add bar-web in the audience-list?

Reply:
I think Bar Web is only using get all fees and fee lookup which don't require an access token.

Reply:
@YasharRahvar - ok. fine. is there any other front end application who is consuming our fees-register-app apart from get all fees?